meteor/server/publications/fastTrack.ts

```diff
@@ -0,0 +1,52 @@
+import { Meteor } from 'meteor/meteor'
+import { logger } from '../logging'
+
+export enum FastTrackObservers {
+	TIMELINE = 'timeline',
+}
+
+const fastTrackObserver: {
+	[key: string]: {
+		useCount: number
+		onData: (data: any) => void
+	}
+} = {}
+
+function getKey(key: string, args: any[]): string {
+	return key + args.join(',')
+}
+
+export function setupFastTrackObserver<T>(
```

Comment:
Missing documentation: this function can only set up 1 observer function per key + keyargs, I presume this is acceptable for the current implementation but can lead to confusion in the future.